src/test/java/org/prebid/server/validation/RequestValidatorTest.java

```diff
@@ -1279,7 +1280,113 @@ public void validateShouldReturnValidationResultWithErrorWhenContentTypeOutOfPos
 
         // then
         assertThat(result.getErrors()).hasSize(1)
-                .containsOnly("request.imp[0].native.request.context must be in the range [1, 3]. Got 100");
+                .containsOnly("request.imp[0].native.request.context is invalid. See https://iabtechlab.com/wp-content/uploads/2016/07/OpenRTB-Native-Ads-Specification-Final-1.2.pdf#page=39");
+    }
+
+    @Test
+    public void validateShouldReturnValidationResultWithErrorWhenContentSubTypeOutOfPossibleValuesRange()
```

Comment:
Please add tests that cover other possible invalid contexType/contextSubtypes combinations and correct handling of null values as well.